projects/plugins/jetpack/_inc/client/earn/index.jsx

```diff
@@ -0,0 +1,202 @@
+	isUnavailableInOfflineMode as isUnavailableInOfflineModeSelector,
+} from 'state/connection';
+import { isAtomicSite as isAtomicSiteSelector } from 'state/initial-state';
+import { getModule } from 'state/modules';
+import { isModuleFound as isModuleFoundSelector } from 'state/search';
+import { Ads } from './ads';
+
+/**
+ * Earn Feature description card.
+ *
+ * @param {object} props - Component props.
+ * @returns {React.Component} Feature description and CTA.
+ */
+function EarnFeatureButton( props ) {
+	const {
+		buttonText,
+		featureConstant = '',
+		featureName,
+		infoLink,
+		infoDescription,
+		supportLink,
+		title,
+	} = props;
+
+	const trackButtonClick = useCallback( () => {
+		analytics.tracks.recordJetpackClick( {
+			target: `visit-${ featureName }`,
+			feature: 'earn',
+		} );
+	}, [ featureName ] );
+
+	return (
+		<SettingsCard
+			{ ...props }
+			header={ title }
+			hideButton
+			module="earn"
+			feature={ featureConstant ? featureConstant : undefined }
+		>
+			<SettingsGroup
+				disableInOfflineMode
+				disableInSiteConnectionMode
+				support={ {
+					link: supportLink,
+				} }
+			>
+				{ infoDescription }
+			</SettingsGroup>
+			<Card
+				compact
+				className="jp-settings-card__configure-link"
+				onClick={ trackButtonClick }
+				href={ infoLink }
+				target="_blank"
+			>
+				{ buttonText }
+			</Card>
+		</SettingsCard>
+	);
+}
+
+/**
+ * Earn Section.
+ *
+ * @param {object} props - Component props.
+ * @returns {React.Component} Earn settings component.
+ */
+function Earn( props ) {
+	const { active, hasConnectedOwner, isModuleFound, isOffline, searchTerm, siteRawUrl } = props;
+
+	const foundAds = isModuleFound( 'wordads' );
+
+	if ( ! searchTerm && ! active ) {
+		return null;
+	}
+
+	const paymentBlocks = () => {
+		if ( isOffline ) {
+			return (
+				<SettingsGroup>
+					<div className="jp-form-block-fade" />
+					{ __( 'Unavailable in Offline Mode.', 'jetpack' ) }
+				</SettingsGroup>
+			);
+		}
+
+		if ( ! hasConnectedOwner ) {
+			return (
+				<SettingsCard
+					{ ...props }
+					hideButton
+					module="earn"
+					header={ __( 'Collect payments', 'jetpack' ) }
+				>
+					<ConnectUserBar
+						feature="earn"
+						featureLabel={ __( 'Collect payments', 'jetpack' ) }
+						text={ __( 'Connect to discover tools to earn money with your site.', 'jetpack' ) }
+					/>
+				</SettingsCard>
+			);
+		}
+
+		return (
+			<>
+				<EarnFeatureButton
+					{ ...props }
+					featureName="payments"
+					title={ __( 'Collect payments', 'jetpack' ) }
+					supportLink={ getRedirectUrl( 'jetpack-support-jetpack-blocks-payments-block' ) }
+					infoLink={ getRedirectUrl( 'wpcom-earn-payments', {
+						site: siteRawUrl,
+					} ) }
+					infoDescription={ __(
+						'Let visitors pay for digital goods and services or make quick, pre-set donations by enabling the Payment Button block.',
+						'jetpack'
+					) }
+					buttonText={ __( 'Enable Payment Button', 'jetpack' ) }
```

Comment:
I'm not sure about Jetpack copy style but you use a lot of "Title Case" instead of "Sentence case". :-) I'd expect more latter e.g. at WP.com

Reply:
I'm not quite sure in this case, since those are feature names and not just names. I'm happy to make the change though. c9b1196b6432e5ccabb1165f17a02bc73134387d